### ISSUE TITLE
Installing aarch64 binaries via pip

### ISSUE THREAD

**Issue by RVSagar** [Simple-Robotics/proxsuite#201]
Hi There,

Thanks for the nice library! I'm currently using it with a humanoid (PAL REEM-C) and TSID. I was wondering whether aarch64 binaries are available for install via pip? I tried to do `pip install proxsuite` on Ubuntu 22.04 ARM64 version but am met with: 

```bash
parallels@docker:~# pip install proxsuite
Defaulting to user installation because normal site-packages is not writeable
ERROR: Could not find a version that satisfies the requirement proxsuite (from versions: none)
ERROR: No matching distribution found for proxsuite

```

If helpful, I'm using Parallels 18 on an M1 MacBook Pro.

Thanks for your time!

**Comment by jcarpent** [comment]
Thanks for your feedback @RVSagar.
We may easily active the support for conda. Could you rely on conda instead of pip? Conda is as simple as pip in terms of usage, if not simpler.

**Comment by ximeng96** [comment]
Hi, I used both conda and pip, and both of them return errors. I used Intel 7 and windows 10. The error returned by conda is as below:
![image](https://user-images.githubusercontent.com/56058520/224932439-af54b3e6-959e-4211-ba3c-58f046f54f75.png)

**Comment by jcarpent** [comment]
@ximeng96 Is your issue related to this one? If not, please open a new issue and provide more details.

PS: from what I've looked, your are missing to work with a clean environment with conda.

**Comment by jcarpent** [comment]
@RVSagar you can already use `conda install proxsuite -c conda-forge` on aarch64.

**Comment by RVSagar** [comment]
That was fast! Thanks @fabinsch and @jcarpent, I can see that `pip install proxsuite` works on my aarch64 system now after #202. And thanks for the note on conda, I see most architectures are supported there.

Have a great day!
